tests/integration/ethernet_mtu_test.py

```diff
@@ -136,12 +149,15 @@ def eth1_up_with_mtu_1900(eth1_up):
     eth1_desired_state = desired_state[Interface.KEY][0]
     eth1_desired_state[Interface.MTU] = 1900
 
-    libnmstate.apply(desired_state)
```

Comment:
If it is possible to change all the `libnmstate.apply()` to `apply_with_description()` in all the integration tests, I can do it in a followup PR.

Reply:
Yes. That's the ultimate goal.